application/frontend/controllers/AuthController.php

```diff
@@ -90,8 +75,17 @@ public function actionLogin()
                 }
             }
 
-            $accessToken = $user->createAccessToken($clientId, User::AUTH_TYPE_LOGIN);
-
+            $accessToken = $user->createAccessToken(User::AUTH_TYPE_LOGIN);
+            $secure = YII_ENV != 'dev' || YII_ENV != 'test' ? true : false;
+
+            \Yii::$app->response->cookies->add(new \yii\web\Cookie([
+              'name' => 'access_token',
+              'value' => $accessToken,
+              'expire' => $user->access_token_expiration,
+              'httpOnly' => true, // Ensures the cookie is not accessible via JavaScript
+              'secure' => $secure,   // Ensures the cookie is sent only over HTTPS
+              'sameSite' => 'Lax', // Adjust as needed
+            ]));
```

Comment:
this could be extracted to a separate function so any future changes are the same on both login and reset flows